src/js/utils/convertStringToObject.js

```diff
@@ -0,0 +1,21 @@
+const convertStringToObject = (string) => {
+  try {
+    const plainText = JSON.stringify(string.replace(/\n/g, ''));
+    const jsonText = plainText
```

Comment:
If we find this stack of replace() calls causes bugs in the future, I'd like us to explore using a community tested library for converting JSON Editor Text to a JSON Object.